Makefile

```
@@ -0,0 +1,40 @@
+
+MAKEFILE_ROOT=$(shell pwd)
+
+# install: Ensure that the Argo CD namespace exists, and that CRDs we are using are applied to the current cluster
+install:
+	kubectl create ns argocd || true
+	kubectl apply -f $(MAKEFILE_ROOT)/backend/config/crd/bases
+	kubectl apply -f $(MAKEFILE_ROOT)/backend-shared/config/crd/bases/managed-gitops.redhat.com_operations.yaml
+	kubectl apply -f https://raw.githubusercontent.com/argoproj/argo-cd/release-2.2/manifests/crds/application-crd.yaml
+
+# start: start all the components
+start:
```

```suggestion
start: download-deps ## Start all the components; compile & run
```

1. Make use of the `##` syntax sugar so the `make help` works as expected.
2. Make sure `goreman` is pre-installed.

Example output:

```
$ make -n start
go install github.com/mattn/goreman@latest
~/go/bin/goreman start
```